src/components/datalist-item/__tests__/DatalistItem.test.tsx

```diff
@@ -1,66 +1,44 @@
 import React from 'react';
 import sinon from 'sinon';
-
+import { shallow } from 'enzyme';
 import DatalistItem from '..';
 
 const sandbox = sinon.sandbox.create();
-
```

Comment:
All the newlines in this file are removed. I think we might want to keep these to separate tests / logical groupings of code and help readability.